Unit1.pas

```diff
@@ -1112,17 +1168,66 @@ function MakeUni(s: ansistring): ansistring;
   Tsopc.CustomSort(Comparestr);
   TsFnc.CustomSort(CompareLabel);
   TsData.CustomSort(CompareLabel);
+
+  if isdc then
+  begin
+    // Clean up remaining trailing nops
+    i := form4.listbox1.items.count - 1;
```

Comment:
trailing NOP are to fill the data, PSO DC and GC need the dada to be aligned to 4 bytes.

So that's why NOP get added when saving. It's ok to clean them on load, just make sure they aren't fully removed when saving else the game will freeze when quest load.